internal/grpcutil/target_test.go

```diff
@@ -70,7 +70,7 @@ func TestParseTargetString(t *testing.T) {
 		// If we can only parse part of the target.
 		{targetStr: "://", want: resolver.Target{Scheme: "", Authority: "", Endpoint: "://"}},
 		{targetStr: "unix://domain", want: resolver.Target{Scheme: "", Authority: "", Endpoint: "unix://domain"}},
-		{targetStr: "unix://a/b/c", want: resolver.Target{Scheme: "unix", Authority: "a", Endpoint: "/b/c"}, wantWithDialer: resolver.Target{Scheme: "unix", Authority: "a", Endpoint: "b/c"}},
+		{targetStr: "unix://a/b/c", want: resolver.Target{Scheme: "unix", Authority: "a", Endpoint: "b/c"}},
```

Comment:
This should be `Endpoint: "/b/c"` since we have the 3-slashes form.  It doesn't matter in practice since the authority is non-empty, so it will be an error either way, but that's how it should be.

Reply:
It's "/b/c" now due to the revert of the +"/" change.